demisto_sdk/commands/validate/validators/PB_validators/PB126_is_default_not_only_condition.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.constants import PlaybookTaskType
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+class IsDefaultNotOnlyConditionValidator(BaseValidator[ContentTypes]):
+    error_code = "PB126"
+    description = (
+        "ensure that conditional tasks have more than path which is not the default one"
+    )
+    rationale = (
+        "We want to ensure that conditional tasks have more than path which is not the default one o/w it "
+        "make no sense to have such"
+    )
+    error_message = (
+        "Playbook task with id:{} only has a default option. Please remove this task or add another "
+        "non-default option to the task"
+    )
+    related_field = "conditions"
+    is_auto_fixable = False
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        validation_results = []
+        for content_item in content_items:
+            for task in content_item.tasks.values():
+                if (
+                    task.type == PlaybookTaskType.CONDITION
+                    and task.message
+                    and task.message.get("replyOptions")
+                ):
+                    reply_options = set(
+                        map(str.upper, task.message.get("replyOptions", []))
+                    )
+                    if len(reply_options) == 1 and "#default#".upper() in reply_options:
+                        validation_results.append(
+                            ValidationResult(
+                                validator=self,
+                                message=self.error_message.format(task.id),
+                                content_object=content_item,
+                            )
+                        )
```

Comment:
I think it's better to save all task ids in a list and then have it formatted into one validation result to save extra traffic in the logs.